multiversx_sdk_cli/cli_wallet.py

```diff
@@ -88,18 +88,20 @@ def setup_parser(args: List[str], subparsers: Any) -> Any:
 
 
 def new_wallet(args: Any):
-    mnemonic = generate_mnemonic()
-    print(f"Mnemonic: {mnemonic}")
-    secret_key, pubkey = wallet.derive_keys(mnemonic)
+    mnemonic = Mnemonic.generate()
+    print(f"Mnemonic: {mnemonic.get_text()}")
+    secret_key = mnemonic.derive_key()
+    pubkey = secret_key.generate_public_key()
+
     if args.pem:
         pem_file = prepare_file(args.output_path, ".pem")
         address = Address(pubkey)
-        pem.write(pem_file, secret_key, pubkey, name=address.bech32())
+        pem.write(pem_file, secret_key.buffer, pubkey.buffer, name=address.bech32())
         logger.info(f"Pem wallet generated: {pem_file}")
     if args.json:
         json_file = prepare_file(args.output_path, ".json")
         password = getpass.getpass("Enter a new password:")
-        save_to_key_file(json_file, secret_key, pubkey, password)
+        save_to_key_file(json_file, secret_key.hex(), pubkey.hex(), password)
```

Comment:
All right. In a future PR, `save_to_key_file` can be replaced by the new functions from `multiversx-sdk-wallet`.